src/main/java/com/tstepnik/planner/domain/Task.java

```diff
@@ -16,13 +18,36 @@
     @Enumerated(EnumType.STRING)
     private Importance importance;
 
-    private boolean isDone;
+    private Boolean isDone;
```

Comment:
Again, why you have these changes here?

Reply:
Because i created that branch before merge feature/mapstruct with develop where we solved that problem.

Reply:
Is mapstruct finally merged in to develop?

Reply:
yes, i pull develop to this branch everything is up to date

Reply:
By "this branch" do you mean `feature/statistics`? When merging branches try to use generated commit messages. They are easier to find. If you dont get generated message use something like: "Merging develop into feature/statistics"

Reply:
Yes. I did "ad missed classes from develop".